docs/design-proposals/multi-tenancy/multi-tenancy-api-resource-model.md

```diff
@@ -159,14 +159,88 @@ Here are some examples:
 
 ### CRD Support
 
- Each tenant can install their CRD independently, without worrying about resource name collisions. The API group/vision/kind (G/V/K) defined by the CRD object is only accessible to that tenant.
+#### Principles and Outlines of Design
+
+ The following Arktos multi-tenancy design principles will be observed in the design of CRD:
  
- For CRDs installed in system world, by default it's only accessible for tenant users. But if it's marked with "multi-tenancy.k8s.io/published" annotation, it will be accessible to all tenants.
+ 1. **isolation**. Each regular tenant can independently create/delete/update their CRDs and custom resources based on the CRDs. There will be NO CRD G/V/K collision between two regular tenants. Actually, the CRDs and custom resources of one regular tenant are invisible to another regular tenant.  
+
+ 2. **Managebility**. A cluster operator (he should be user under the system tenant) can introduce a new CRD, where he can choose to:
+  * make it usable only to the system tenant.
+  * or, make it usable only to a spefic set of to regular tenants and the system tenants, 
+  * or, usable to all the regular tenants. 
+  * Additionally, the applicable range of a CRD can be changed by the cluster operator at any time in a simple and quick manner. That will be useful to CRD test and rolling update.  
  
- If there is a collision between CRDs in system world and tenant world, the latter one takes precedence if it's accessed by a tenant.
+ Note: it does not contradict the principle of isolation. The isolation principle is about two regular tenants, while this principle is about the system tenant and regular tenants. 
+
+ 3. **Autonomy**. The tenant admin has the final say on what CRD will be used in his tenant space when there is an overlap between the tenant's CRD and that of the system tenant. The tenant can choose MultiTenancyCRD policy to be :
```

Comment:
Autonomy is also about a tenant can install his own CRDs without help from cluster admins.

Reply:
Yep, actually this point is more point. Added it as the first line in the subsection of "autonomy".